Sources/FluentKit/Model/AnyModel.swift

```diff
@@ -46,20 +48,43 @@ extension AnyModel {
     }
 
     var anyID: AnyID {
-        guard let id = Mirror(reflecting: self).descendant("_id") as? AnyID else {
-            fatalError("id property must be declared using @ID")
+        let mirror = Mirror(reflecting: self)
+        let children = mirror.children
+        
+        let table = idTable.currentValue ?? AnyIDTable()
+        let value: Any
+        
+        if let index = table.ids.first(where: { $0.typeId == ObjectIdentifier(Self.self) })?.1 {
+            value = children[index].value
+        } else {
+            guard let index = children.firstIndex(where: { $0.label == "_id" }) else {
+                fatalError("id property must be declared using @ID")
+            }
+            
+            value = children[index].value
         }
-        return id
+        
+        assert(value is AnyID, "id property did not conform to AnyID, make sure it's declared using @ID")
+        
+        return value as! AnyID
     }
 }
 
+private final class AnyIDTable {
+    var ids: [(typeId: ObjectIdentifier, offset: AnyIndex)] = []
```

Comment:
I think we discussed this at ServerSide.swift, but it might be worth adding a note to the PR or comment in the code explaining why you are using a tuple array instead of a dictionary.

Reply:
@calebkleveter I will. I can understand the need for a comment :)